jni/prepare_ml-api.sh

```diff
@@ -12,44 +12,46 @@
 
 TARGET=$1
 # Note: zip name can be nnstreamer-native-*.zip but this file is heavier to download
-FILE_PREFIX=nnstreamer-single-native
+FILE_PREFIX=nnstreamer-native
```

Comment:
how about nnstreamer-lite-native? it contains the header we need and requires smaller download?

Reply:
Yup, If there is a problem we can revert :)